odo/backends/csv.py

```diff
@@ -91,7 +92,10 @@ def append_dataframe_to_csv(c, df, dshape=None, **kwargs):
     encoding=kwargs.get('encoding', c.encoding)
 
     if ext(c.path) in compressed_open:
-        f = compressed_open[ext(c.path)](c.path, mode='a')
+        kwargs = dict(mode='at')
+        if sys.version_info[0] >= 3:
```

Comment:
alas, we have to support python 2.6, which doesn't implement `sys.version_info` as a `namedtuple`:

```python
Python 2.6.9 |Continuum Analytics, Inc.| (unknown, Jan 10 2014, 13:33:57)
[GCC 4.2.1 (Apple Inc. build 5577)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.version_info
(2, 6, 9, 'final', 0)
```